src/main/kotlin/org/lsst/dax/albuquery/tasks/QueryTask.kt

```diff
@@ -63,8 +64,10 @@ class QueryTask(
 ) : Callable<QueryTask> {
 
     val columnAnalyzer: TableAndColumnExtractor
+    val phaseInfo: PhaseInfo
 
     init {
+        phaseInfo = PhaseInfo(phase = "READY")
```

Comment:
I don't know where this comment went before, but I believe from looking at the spec, that the first state is PENDING, and that READY isn't a state in the state machine.

http://www.ivoa.net/documents/UWS/20161024/REC-UWS-1.1-20161024.pdf
Section 2.1.3. Execution Phase